### ISSUE TITLE
Which one should be specified in mix_json in debug.yaml?

### ISSUE THREAD

**Issue by srdfjy** [facebookresearch/svoice#42]
Hi，I have generated the following data set, the speakers in train, valid, and test are all independent，which one should be specified in mix_json in debug.yaml?

egs/test/
├── cv
│   ├── mix.json
│   ├── s1.json
│   └── s2.json
├── tr
│   ├── mix.json
│   ├── s1.json
│   └── s2.json
└── tt
    ├── mix.json
    ├── s1.json
    └── s2.json

**Comment by adiyoss** [comment]
Hi @srdfjy,
The `mix_json` parameter in the `debug.yaml` (or any other dataset config) will be used for separating mixtures every 10 epochs (you can modify it using the `eval_every` parameter in the config file), so you can listen to the samples. 
If your validation set is not so big you can use the `mix.json` file from the `cv` folder. In case your validation set is big, I suggest creating a small subset of it and use that for the `mix_json` param, so it won't make the training process much slower.

**Comment by srdfjy** [comment]
@adiyoss thanks
